cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -32,10 +32,19 @@ class TowerVariableValue(models.Model):
         inverse="_inverse_is_global",
         store=True,
     )
-
-    value_char = fields.Char(string="Value")
     note = fields.Text(related="variable_id.note", readonly=True)
     active = fields.Boolean(default=True)
+    variable_type = fields.Selection(
+        selection=[("s", "String"), ("o", "Options")],
+        string="Variable Type",
+        related="variable_id.variable_type",
+        store=True,
```

Comment:
Do we really need to store this field? Just wondering, because as far as it can be seen from the code it's used only for domain in view.